src/frequenz/sdk/microgrid/component/_component.py

```diff
@@ -2,16 +2,19 @@
 # Copyright © 2022 Frequenz Energy-as-a-Service GmbH
 
 """Defines the components that can be used in a microgrid."""
+from __future__ import annotations
 
 from dataclasses import dataclass
 from enum import Enum
+from typing import TYPE_CHECKING
 
 import frequenz.api.common.components_pb2 as components_pb
 import frequenz.api.microgrid.grid_pb2 as grid_pb
 import frequenz.api.microgrid.inverter_pb2 as inverter_pb
 
-from ...timeseries import Current
-from ..fuse import Fuse
+if TYPE_CHECKING:
+    # Break circular import by enclosing these type hints in a `TYPE_CHECKING` block.
```

Comment:
Super silly, but the comment seems a bit too redundant.

```suggestion
    # Break circular import
```

Reply:
Added a new commit to amend this comment and similar ones